WalletWasabi.Tests/UnitTests/Crypto/TranscriptTests.cs

```diff
@@ -158,7 +158,7 @@ public void SyntheticNoncesVectorTest(int size)
 			var transcript = new Transcript(protocol);
 			var secretNonceProvider = transcript.CreateSyntheticSecretNonceProvider(witness, rnd);
 
-			var secretNonce = secretNonceProvider.Sequence.First();
+			var secretNonce = secretNonceProvider.GetScalarVector();
```

Comment:
Is it actually good idea to use `var` here? I believe it makes sense to use `var` when the line is in form: `var variable = new Type(args);` other than that it means that whoever reads the code either believes you or has to check types. I think we should avoid that in crypto code (actually even in the remaining code but that's for another discussion).

I believe that VSCode has also that feature that you simply write `var variable = something;` and click CTRL + `.` on `var` to convert it to explicit type.